src/api/middleware/validators/offer.js

```diff
@@ -20,19 +29,17 @@ const create = useExpressValidators([
 
     body("publishDate", ValidationReasons.DEFAULT)
         .optional()
-        .isISO8601({ strict: true }).withMessage(ValidationReasons.DATE).bail()
-        .toDate(),
+        .isISO8601({ strict: true }).withMessage(ValidationReasons.DATE).bail(),
 
     body("publishEndDate", ValidationReasons.DEFAULT)
         .exists().withMessage(ValidationReasons.REQUIRED).bail()
         .isISO8601({ strict: true }).withMessage(ValidationReasons.DATE).bail()
         .isAfter().withMessage(ValidationReasons.DATE_EXPIRED).bail()
         .custom((publishEndDateCandidate, { req }) => {
-            const { publishDate: publishDateRaw } = req.body;
+            const { publishDate: publishDateCandidate } = req.body;
+            // publishDateRaw will be a Date instance because it's sanitized on the publishDateValidator
```

Comment:
remove comment, since publishDateRaw does not exist, remember to check dangling comments always, as they do more harm than good